components/studio/bin/hab-studio.ps1

```diff
@@ -254,6 +254,16 @@ function New-Studio {
     $env:HAB_CACHE_KEY_PATH = Join-Path $env:FS_ROOT "hab\cache\keys"
   }
 
+  $env:HAB_CACHE_SSL_PATH = Join-Path $env:FS_ROOT "hab\cache\ssl"
+  if (!(Test-Path $env:HAB_CACHE_SSL_PATH)) {
+    mkdir $env:HAB_CACHE_SSL_PATH | Out-Null
+  }
+
+  $sslCacheSourcePath = Join-Path $env:SystemDrive "hab\cache\ssl"
+  if (Test-Path $sslCacheSourcePath) {
+    Write-HabInfo "Populating SSL certificate cache at $env:HAB_CACHE_SSL_PATH"
+    Copy-Item -Path $sslCacheSourcePath\* -Destination $env:HAB_CACHE_SSL_PATH
+  }
```

Comment:
Why was the `SSL_PATH` environment variable needed? It doesn't seem to be referenced in this code.

Reply:
ARTIFACT_PATH and CERT_PATH are currently only used in the Linux Studio.  We don't have exact parity between Windows and Linux Studios atm.

Reply:
If this PR is all about Windows, and Windows doesn't use `CERT_PATH`, why were the Rust changes above to add `CERT_PATH` required for this?

Reply:
The CERT_PATH changes are not strictly required for Windows (atm) - those look like a missed use case to supplement the Linux PR that was done earlier.  If and when we bring Windows into parity with Linux for the env, we will be able to leverage those for Windows also.

Reply:
In that case, it might be nice to separate these into different PRs.

Reply:
OK

Reply:
Well nvm, I think this is going to be needed for the Windows Docker Studio scenario after all - specifically, in the error case in enter.rs (where there is no CERT_PATH env defined) we set the env var, and then in docker.rs we use that var for the volume mount.